src/app/lib/logger.node.js

```diff
@@ -47,18 +47,75 @@ const folderAndFilename = name => {
   return fileparts.splice(-2).join(path.sep);
 };
 
-const logger = callingFile => {
+const logToFile = callingFile => {
   createLogDirectory(LOG_DIR);
 
   return createLogger({
     format: combine(
       label({ label: folderAndFilename(callingFile) }),
       simple(),
-      timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+      timestamp({ format: 'YYYY-MM-DD HH:mm:ss.SSS' }),
       customFormatting,
     ),
     transports: [fileTransport, consoleTransport],
   });
 };
 
+const logEventMessage = (event, message) => {
+  const logObject = {
+    event,
+    message,
+  };
+
+  return JSON.stringify(logObject, null, 2);
+};
+
+class Logger {
+  constructor(callingFile) {
+    const fileLogger = logToFile(callingFile);
+
+    this.error = (event, message) => {
+      if (message) {
+        fileLogger.error(logEventMessage(event, message));
+      } else {
+        fileLogger.error(event);
+      }
+    };
```

Comment:
Instead of having two formats for logging, I think it'd be better to enforce always passing in an event name as the first argument with the optional object of params to be passed in as the second argument.